docs/dashboards-new/index.md

```diff
@@ -86,8 +86,14 @@ In this section, we'll introduce the following concepts:
 </div>
 <div className="box smallbox13 card">
   <div className="container">
-  <a href="/docs/dashboards-new/share-dashboard-new/"><img src={useBaseUrl('img/icons/dashboards.png')} alt="icon" width="40"/><h4>Sharing Dashboards</h4></a>
+  <a href="/docs/dashboards-new/share-dashboard-new"><img src={useBaseUrl('img/icons/dashboards.png')} alt="icon" width="40"/><h4>Sharing Dashboards</h4></a>
   <p>Learn how to save share a custom dashboard with colleagues.</p>
```

Comment:
Could you correct this sentence? remove "save"

Reply:
Updated.